06_create_cluster.sh

```diff
@@ -64,6 +64,14 @@ if [[ -n "${APPLY_EXTRA_WORKERS}" ]]; then
     fi
 fi
 
+# Mirrored OLM operators need to have respective ICSP and CatalogSources applied. This can be done
```

Comment:
This works, but the other option is to copy the manifests into $OCP_DIR/manifests so they're included in the deployment by the installer. That's how the old version handled this (line 138 and 139 of the 04 script).

Reply:
Oh okay, I need to try it out then. I just didn't know about this "auto-apply mechanism" so I took the manual path